library/oneview_hypervisor_cluster_profile.py

```diff
@@ -158,6 +167,12 @@ def __update(self):
         if "newName" in self.data:
             self.data["name"] = self.data.pop("newName")
 
+        # Below code will get the virtual switch layout and modifies the update body if virtualswitches key is present in json data
+        if self.create_vswitch_layout and self.data.get('hypervisorHostProfileTemplate') and \
```

Comment:
To avoid code duplication, move this logic to a private method, and call it for create and update